merfi/tests/test_iso.py

```diff
@@ -0,0 +1,21 @@
+import os
+import pytest
+import subprocess
+from merfi.iso import Iso
+
+class TestIso(object):
+
+    def setup(self):
+        self.iso = Iso([])
+
+    def test_sha256sum(self, tmpdir):
+        f = tmpdir.join('test.iso')
+        f.write('ISOCONTENTS')
+        self.iso.output = str(f)
+        self.iso.make_sha256sum()
+        checksum_file = self.iso.output + '.SHA256SUM'
+        with open(checksum_file, 'r') as shaf:
+            assert shaf.read() == "d8d322f6864229f8c9ef1b0845dd9e182c563c508fec30618fdb9b57c70a0147  test.iso\n"
+            # ... and validate checksum_file's syntax with sha256sum:
+            os.chdir(os.path.dirname(checksum_file))
+            assert subprocess.call(['sha256sum', '-c', checksum_file]) == 0
```

Comment:
The reason I have this here is that it functionally validates what a user would do with this ISO's checksum file: they'd run `sha256sum -c` on it. In fact this uncovered a bug in my implementation: originally I'd reversed the order of the filename and the sha checksum in the file, so `sha256sum -c` didn't recognize the file as valid.

It's more of a functional test than a unit test. I'll expand the comment in the code in order to explain this further, and if you still don't like it we can just remove it.